traffic_ops/install/bin/_postinstall.pl

```diff
@@ -1,919 +0,0 @@
-#!/usr/bin/perl
```

Comment:
While it's true that the `master` branch does not use `_postinstall.pl`, that does not mean that `_postinstall.pl` itself is unused. In ATC 6, it was deprecated, pending removal. So it should be removed in its own PR, rather than in a PR of things that are totally unused.